docs/faq/questions/cloud-faq.mdx

```diff
@@ -267,18 +267,19 @@ Cloud.
 - `https://assets.cypress.io` - **Asset CDN** (Org logos, icons, videos,
   screenshots, etc.)
 - `https://authenticate.cypress.io` - **Authentication API**
-- `https://capture.cypress.io` - **Cypress Test Replay**
```

Comment:
Can we leave this line in place?

Reply:
This is being added when the v13 docs branch is deployed. It was added here incorrectly.